lib/steep/diagnostic/ruby.rb

```diff
@@ -286,7 +286,13 @@ class NoMethod < Base
         def initialize(node:, type:, method:)
           loc = case node.type
                 when :send
-                  node.loc.selector
+                  loc = node.loc
+
+                  if loc.respond_to?(:operator)
```

Comment:
Using `respond_to?` as the Parser gem recommends.

https://github.com/whitequark/parser/blob/master/lib/parser/source/map.rb#L44-L48